action.yml

```
@@ -15,6 +15,11 @@ inputs:
     description: GitHub Token used to commit metrics
     default: ""
 
+  # Branch to commit
+  committer_branch:
+    description: The branch used to commit metrics
+    default: main_branch
```

```suggestion
    default: ""
```
This is actually the default value passed by GitHub to action, so you need to leave it empty so it'll defaults on `github.context.ref.replace(/^refs[/]heads[/]/, "")` in the action 🙂